tiny/__main__.py

```diff
@@ -34,8 +31,6 @@ def sig_cb(display, sig_num, frame):
 
 def main(argv) -> None:
     with Display() as display:
-        signal.signal(signal.SIGINT, partial(sig_cb, display))
```

Comment:
I think we can remove the `sig_cb` function if we don't need that here.

Reply:
Done!